src/App/use-check-updates.ts

```diff
@@ -0,0 +1,27 @@
+import { message } from 'antd';
+import { useEffect, useState } from 'react';
+
+export function useCheckUpdates(): void {
+  const [, setCheckedUpdates] = useState(false);
```

Comment:
Why do we need this state? I think it's not required. The empty dependencies `[]` in the `useEffect` means that it will be called only once during app mount.